Tests/Unit/MiniAppDownloaderTests.swift

```diff
@@ -84,8 +84,8 @@ class MiniAppDownloaderTests: QuickSpec {
                      let oldMiniAppDirectory = FileManager.getMiniAppVersionDirectory(with: appId, and: "\(versionId).1")
                      var isDir: ObjCBool = true
 
-                     expect(FileManager.default.fileExists(atPath: miniAppDirectory.path, isDirectory: &isDir)).toEventually(equal(true), timeout: .seconds(10))
-                     expect(FileManager.default.fileExists(atPath: oldMiniAppDirectory.path, isDirectory: &isDir)).toEventually(equal(false), timeout: .seconds(10))
+                     expect(FileManager.default.fileExists(atPath: miniAppDirectory.path, isDirectory: &isDir)).toEventually(equal(true), timeout: .seconds(30))
```

Comment:
30s seems really long 😣 Is is possible to mock FileManager on iOS? Maybe in the future we can try that?

Reply:
That's a max timeout for that testcase anyway, it will pass immediately if the expected value is returned.